pom.xml

```diff
@@ -16,7 +16,9 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.version>3.5.0</maven.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <web3j.version>3.3.1</web3j.version>
+        <web3j.version>3.4.0</web3j.version>
+        <org.ethereum.solcJ>0.4.23</org.ethereum.solcJ>
```

Comment:
the unix version inside this library version is broken

Reply:
I do not understand when you say with "unix version" and for what library.

Reply:
the tests on travis are failing, because of the library "ethereum solcJ". In the version 0.4.23 the solc compiler for unix is broken.